src/Fixer/PhpUnit/PhpUnitStrictFixer.php

```diff
@@ -74,7 +74,7 @@ public function testSomeTest()
                 ),
             ],
             null,
-            'Risky when any of the functions are overridden.'
+            'Risky when any of the functions are overridden, or you are testing object equality.'
```

Comment:
Thanks for the fast and nice proposal!
I would drop the Oxford Comma and rephrase to;
 `'Risky when any of the functions are overridden or when testing object equality.'`
just to stay in the same style of the rest of most of the README, we kind of like to have the same style with everything here, not stating one is better than the other ^_^

Reply:
Fixed

> we kind of like to have the same style with everything here

That's kind of the whole point of `php-cs-fixer` right? :smile: 

How can I tell which is the lowest maintained branch?

Reply:
The lowest maintained branch is 2.2 currently. I think we don't have any documentation about that.